vendor/engines/secure_rooms/README.md

```diff
@@ -1,2 +1,57 @@
 # SecureRooms
 
+The SecureRooms engine adds a product type supporting an access-restricted,
+bill-by-time room.
+
+## Installation
+
+Make sure the `gem "secure_rooms",   "~> 0.0.1", path: "vendor/engines/secure_rooms"`
```

Comment:
The `"~> 0.0.1"` isn't necessary, and we don't have it in our gemfile. Since it's on the app path, you're always getting the latest version no matter what.

In theory, we should get rid of all the version checks for our engines.